image/templates/helm/stackrox-secured-cluster/README.md.htpl

```diff
@@ -86,40 +71,6 @@ helm install -n stackrox --create-namespace \
   or the name of an existing cluster, in which case the name will be reused and associated
   with the Kubernetes cluster on which the chart is installed.
 
-To access StackRox Docker images, you also need image pull credentials. While
```

Comment:
Same suggestions as for the Central chart readme:
1. Optionally keep this block about image pull credentials.
2. Double-check `helm install ...`/`helm upgrade ...` commands against the fresh ones from here https://github.com/stackrox/stackrox/#quick-installation-via-helm

Reply:
Done!